src/pca_report_library/customer/graphs.py

```diff
@@ -38,88 +52,90 @@ def graph_builder(assessment_ID, labels):
             success = False
 
     if success:
-        print("\tGenerating Graphs for " + assessment_ID + " Report...")
+        print("\tGenerating Graphs for " + assessment_id + " Report...")
         # Loads report data into a dictionary from json file.
-        with open(dataFile) as f:
-            reportData = json.load(f)
-        reportData["figures"] = list()
+        with open(data_file) as out_file:
+            report_data = json.load(out_file)
+        report_data["figures"] = list()
 
         COLOR = "#101010"
         mpl.rcParams["text.color"] = COLOR
         mpl.rcParams["axes.labelcolor"] = COLOR
         mpl.rcParams["xtick.color"] = COLOR
         mpl.rcParams["ytick.color"] = COLOR
 
-        unique_user_click_rate_vs_report_rate_per_level_deception(reportData)
-        timeline_unique_user_clicks_all_levels(reportData)
-        time_first_click_report(reportData)
-        click_rate_based_deception_indicators(reportData)
-        unique_total_click_report_results_by_level(reportData)
-        breakdown_multiple_clicks_by_level(reportData)
-        clicking_user_timeline(reportData)
+        unique_user_click_rate_vs_report_rate_per_level_deception(report_data)
+        timeline_unique_user_clicks_all_levels(report_data)
+        time_first_click_report(report_data)
+        click_rate_based_deception_indicators(report_data)
+        unique_total_click_report_results_by_level(report_data)
+        breakdown_multiple_clicks_by_level(report_data)
+        clicking_user_timeline(report_data)
 
         if labels:
-            count_unique_clickes_by_office_per_level(reportData)
-            percentage_office_clicked_by_level(reportData)
-            count_unique_clicks_by_level_per_office(reportData)
-            perc_total_unique_clicks_belong_office_by_level(reportData)
+            count_unique_clickes_by_office_per_level(report_data)
+            percentage_office_clicked_by_level(report_data)
+            count_unique_clicks_by_level_per_office(report_data)
+            perc_total_unique_clicks_belong_office_by_level(report_data)
 
-        with open("reportData_" + reportData["RVA_Number"] + ".json", "w") as fp:
-            json.dump(reportData, fp, indent=4)
+        with open("reportData_" + report_data["RVA_Number"] + ".json", "w") as fp:
+            json.dump(report_data, fp, indent=4)
 
     return success
 
 
-def time_ticks(x, pos):
-    return str(timedelta(seconds=int(x))).split(".")[0]
+def time_ticks(in_val, _):
```

Comment:
If it isn't being used it shouldn't be here.

```suggestion
def time_ticks(in_val):
```